docs/source/distributed.tensor.parallel.rst

```diff
@@ -53,6 +53,9 @@ used for input/output preparation:
 .. autofunction::  make_output_shard_1d
 .. autofunction::  make_output_tensor
 
+.. autofunction::  PrepareModuleInput
```

Comment:
Add it here to pass CI first, Will have more doc changes in the next PR https://github.com/pytorch/pytorch/pull/111176